packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -14,6 +15,8 @@ import 'theme.dart';
 // http://www.google.com/design/spec/layout/metrics-keylines.html#metrics-keylines-keylines-spacing
 const double _kSize = 56.0;
 const double _kSizeMini = 40.0;
+const Duration _kFloatingActionButtonChildSegue = const Duration(milliseconds: 200);
+const Interval _kFloatingActionButtonChildSegueInterval = const Interval(0.25, 1.0);
```

Comment:
Give that this constants are local to this file, we can probably shorten the names to just kSegueDuration and kSegueInterval